docs/built-in-functions.rst

```diff
@@ -181,15 +181,15 @@ Returns ``sha256`` (SHA2 256bit output) hash of input.
 --------
 ::
 
-  def sqrt(a: decimal) -> decimal:
+  def sqrt(a: decimal) -> b:
```

Comment:
This one is actually okay as is. Here `decimal` is the type that `sqrt` returns, according to it's function signature.